tex/chProofs.tex

```diff
@@ -1516,17 +1582,18 @@ \subsection{Using lemmas in proofs}
 Lemma dvdn_addr m d n : d %| m -> (d %| m + n) = (d %| n).
 \end{coq}
 This is a conditional equivalence, expressed as a conditional identity.
-We can replace our current goal with \C{\~\~ (p \%| 1)} by rewriting (the
+We can replace our current goal with \C{\~\~ (p \%| 1)} by rewriting
```

Comment:
I'm having trouble parsing the "by rewriting using" part. But, maybe it's just me at the end of the day. Sorry, if it's a false alert.